wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -159,6 +159,24 @@ public extension ZMConversation {
         return context.executeFetchRequestOrAssert(request) as? [ZMConversation] ?? []
     }
 
+    static func fetchConversationsWithMLSGroupStatus(
+        mlsGroupStatus: MLSGroupStatus,
+        in context: NSManagedObjectContext
+    ) -> [ZMConversation] {
+        let request = Self.fetchRequest()
+
+        let matchingGroupStatus = NSPredicate(
+            format: "%K == \(mlsGroupStatus.rawValue)",
+            argumentArray: [Self.mlsStatusKey]
+        )
+
+        request.predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
+            matchingGroupStatus, .isMLSConversation
+        ])
+
+        return context.executeFetchRequestOrAssert(request) as? [ZMConversation] ?? []
```

Comment:
Can we use replace this Objc-C version with assertions by Swift using `throws` here? `func fetch(_ request: NSFetchRequest<NSFetchRequestResult>) throws -> [Any]`

Reply:
I'll try

Reply:
Done